Assignment5/inc/Car.h

```diff
@@ -0,0 +1,28 @@
+#ifndef CAR_H
+#define CAR_H
+
+#include <string>
+
+class CarAccident 
+{
+public:
+    CarAccident(const std::string& message);
+    const char* what() const noexcept;
+private:
```

Comment:
Car.h file is defining classes, whereas Car.cpp file is implementing the functions of the declared classes. Can we rename the file names so that it is more understandable that what it is actually doing?

Reply:
In C++, the headerFile and the Cpp file should have same name.